deployment/ansible/templates/epoch.yaml.j2

```diff
@@ -1,8 +1,21 @@
 ---
+sync:
+    port: {{ config.sync.port }}
+
 peers:
-    {% for host in groups[hosts_group] | difference([inventory_hostname]) %}
-- "http://{{ hostvars[host]['ansible_default_ipv4']['address'] }}:3013/"
-    {% endfor %}
+{% for host in groups[hosts_group] | difference([inventory_hostname]) %}
+{% if hostvars[host]['sync_public_key'] is defined %}
```

Comment:
@dincho It would be nice to have an improvement to the deployment script so to avoid that for some glitch/typo no node in deployment has peer key.